### ISSUE TITLE
change theme via sequence

### ISSUE THREAD

**Issue by elig0n** [mintty/mintty#1136]
is it possible to change terminal themes via ctrl sequences ?

I know how to do it via keys i.e. : alt+space -> o -> alt+t -> up/down

**Comment by mintty** [comment]
No, but maybe the [`mintheme` script](https://github.com/mintty/mintty/wiki/Tips#using-colour-schemes-themes) supports your use case.
